src/System.IO.Pipelines/System/IO/Pipelines/PositionOperations.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.IO.Pipelines
 {
-    public static class ReadCursorOperations
+    public static class PositionOperations
```

Comment:
Why is it moved back to pipelines?

Reply:
Well, it's pipelines specific. It does not work with an arbitrary position. It only works with positions of IMemoryList, and possibly other things that ReadOnlyBuffer supports.

Reply:
It works with everything that `ReadOnlyBuffer` supports that's why it's in buffers. There is no need to have another copy in Pipelines.

Reply:
Position is more general than Buffers. Look in System.Collections.Sequences. There is a Hashtable and ArrayList implementation that uses it.

Reply:
Then move these methods to be on ROB

Reply:
Hmmm, I have done it initially and it did not work. But now it does. I will fix.

Reply:
I removed it and added InternalsVisible from Pipelines.Tests to ReadOnlyBuffer. We need to design proper seek APIs though.

Reply:
Wait, why do you need `InternalsVisisbleTo` to `Pipelines.Tests`? Just don't move `SeekTests.cs` from primitives.